airflow/providers/google/cloud/example_dags/example_life_sciences.py

```diff
@@ -0,0 +1,108 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Example Airflow DAG that displays interactions with Google Cloud Life Sciences.
+
+This DAG relies on the following OS environment variables:
+
+* GCP_PROJECT_ID - Google Cloud Project to use for the Cloud Function.
+* GCP_GCS_BUCKET - Google Cloud Storage Bucket to use
+* GCP_LIFE_SCIENCES_LOCATION - The Location of the Google Cloud Project
```

Comment:
```suggestion
```
We do not need variable lists because they are very difficult to maintain in good condition. In this case, one variable is missing.